src/main/java/models/Employee.java

```diff
@@ -0,0 +1,56 @@
+package models;
+
+import java.sql.Timestamp;
+
+public class Employee {
+
+  private long employeeId;
+  private String firstName;
+  private String lastName;
+  private String addedDate;
+  private String employmentStatus;
+
+  public long getEmployeeId() {
+    return employeeId;
+  }
+
+  public void setEmployeeId(long employeeId) {
+    this.employeeId = employeeId;
+  }
+
+  public String getFirstName() {
+    return firstName;
+  }
+
+  public void setFirstName(String firstName) {
+    this.firstName = firstName;
+  }
+
+  public String getLastName() {
+    return lastName;
+  }
+
+  public void setLastName(String lastName) {
+    this.lastName = lastName;
+  }
+
+  public String getAddedDate() {
+    return addedDate;
+  }
+
+  public void setAddedDate(String addedDate) {
+    this.addedDate = addedDate;
+  }
+
+  public String getStatus() {
+    return employmentStatus;
+  }
+
+  public void setStatus(String employmentStatus) {
+    this.employmentStatus = employmentStatus;
+  }
+
+  public String toString(){
```

Comment:
- Missing @Override
- Use `StringBuilder` to construct the string instead of having multiple string concatenations. For each concat it will create a new immutable String object in memory and waste space, using `StringBuilder` is more efficient